drivers/openxr/SCsub

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+Import("env")
+
+is_supported_on_platform = True
+if env["platform"] == "android":
+    # may need to set OPENXR_ANDROID_VERSION_SUFFIX
+    env.AppendUnique(CPPDEFINES=["XR_OS_ANDROID", "XR_USE_PLATFORM_ANDROID"])
+
+    # may need to include java parts of the openxr loader
+
+    # temporarily disabled until we finish this logic
+    is_supported_on_platform = False
```

Comment:
@BastiaanOlij Can you create an issue and assign it to me for the Android portion.

Reply:
Shall do :)

Reply:
https://github.com/godotengine/godot/issues/57498